### ISSUE TITLE
[Accessibility]: Code monitoring: getting started page screen reader issues

### ISSUE THREAD

**Issue by almeidapaulooliveira** [sourcegraph/sourcegraph#35148]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34193

### Problem description

When the user goes to the create code monitor flow, the screen reader (VoiceOver, MacOS) says:
'You are currently on the text field.' 

The user can't know if the page changed or what page the system is.

### Expected behavior

The system should inform:
- What page the user is.
- Describe what the text field does: 
   'Give it a short, descriptive name to reference events on Sourcegraph and in notifications.' 

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by umpox** [comment]
Fixed through https://github.com/sourcegraph/sourcegraph/pull/35413
